public/css/customv600.css

```diff
@@ -18,12 +26,16 @@
     color:#555555;
 }
 
-.navMove {
-    display:flex;
-    justify-content:center;
-    align-items:center;
+.navItem {
+    display: grid;
+    justify-content: center;
+    align-items: center;
 }
 
+.navMove:hover {
+    background-color: #dcdcdc;
+    cursor: pointer;
+}
```

Comment:
With the new changes, the table stands out a lot more and seems distracting. I tried making the borders lighter using dev tools and it looks much better. let me know what you think. the border color i used was `#f1f1f1`
**Before**
![image](https://user-images.githubusercontent.com/13910458/102683735-09c20500-4188-11eb-905a-85c5455b3762.png)

**After**
![image](https://user-images.githubusercontent.com/13910458/102683770-560d4500-4188-11eb-8cdf-fbf52ed8599a.png)

Reply:
I've resolved this now, also made it darker in dark mode to be less distracting.